lessons/intro/argparse/index.md

```diff
@@ -0,0 +1,377 @@
+argparse
+========
+
+Knihovna `argparse` je ve standardní knihovně a slouží k vytváření rozhraní pro příkazovou řádku
+(angl. *command line interface*, CLI).
+
+`argparse` toho neumí tolik jako [`click`]( {{ lesson_url('intro/click')}} ), ale pro spoustu
+případů bohatě poslouží a každý s Pythonem ji má k dispozici bez nutnosti další instalace.
+
+## Argumenty příkazové řádky
+
+`argparse` není ve standardní knihovně jediným nástrojem na zpracování argumentů příkazové řádky.
+Mezi jeho hlavní „konkurenty“ patří: [os.environ], [optparse] — předchůdce `argparse` — a [getopt].
```

Comment:
`os.environ` nepracuje s CLI argumenty.

Reply:
Máš recht, ovlivnit chování s tím lze skrze proměnné, ale argumenty to vskutku nejsou. Smazáno.

Mimochodem, text lekce vycházel z lekce o clicku, abych ukázal alternativu k němu, a v ní je tento odstavec:

> Nástroje na zpracování argumentů z CLI jsou i přímo ve standardní knihovně, a dokonce jich není málo: os.environ, argparse, optparse, getopt. S knihovnou click je ale práce mnohem příjemnější a výsledky většinou lépe odpovídají zavedeným konvencím.

Možná by stálo za to jej přeformulovat také.

Reply:
Jej! Díky, opravuju v https://github.com/pyvec/naucse.python.cz/pull/654